src/gameboy/APU/APU.h

```diff
@@ -67,53 +68,53 @@ class APU {
     APUState* getAPUState();
 
 private:
-    uint8_t NR10;
-    uint8_t NR11;
-    uint8_t NR12;
-    uint8_t NR13;
-    uint8_t NR14;
-
-    uint8_t NR21;
-    uint8_t NR22;
-    uint8_t NR23;
-    uint8_t NR24;
-
-    uint8_t NR30;
-    uint8_t NR31;
-    uint8_t NR32;
-    uint8_t NR33;
-    uint8_t NR34;
+    uint8_t NR10{};
+    uint8_t NR11{};
+    uint8_t NR12{};
+    uint8_t NR13{};
+    uint8_t NR14{};
+
+    uint8_t NR21{};
+    uint8_t NR22{};
+    uint8_t NR23{};
+    uint8_t NR24{};
+
+    uint8_t NR30{};
+    uint8_t NR31{};
+    uint8_t NR32{};
+    uint8_t NR33{};
+    uint8_t NR34{};
```

Comment:
Why are you putting `{}` behind each variable declaration?
I recognize this from initializing structs with no data, but I do not think this should be used for primitive variables.

Reply:
Was a suggestion from the IDE complaining that those fields were not initialized by the constructor.

Reply:
Ah, ok